src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -70,8 +70,8 @@ describe 'ProductTypeImporter', ->
       sphereClient = new SphereClient options
 
   beforeEach ->
-    sphereClient.productTypes.fetch()
-    .then (res) ->
+    sphereClient.productTypes
+    .process (res) ->
```

Comment:
If you work with .process it might be good idea to use concurrency limit for Promise.map below too.